src/incubator/expandableOverlay/index.tsx

```diff
@@ -39,18 +50,32 @@ interface ExpandableOverlayMethods {
 }
 
 const ExpandableOverlay = (props: ExpandableOverlayProps, ref: any) => {
-  const {children, expandableContent, useDialog, modalProps, dialogProps, showTopBar, topBarProps, ...others} = props;
+  const {
+    children,
+    expandableContent,
+    useDialog,
+    modalProps,
+    dialogProps,
+    showTopBar,
+    topBarProps,
+    renderCustomOverlay,
+    ...others
+  } = props;
   const [expandableVisible, setExpandableVisible] = useState(false);
   const showExpandable = useCallback(() => setExpandableVisible(true), []);
   const hideExpandable = useCallback(() => setExpandableVisible(false), []);
```

Comment:
We need to invoke the Modal/Dialog's 'onDismiss' in 'hideExpandable' so the user can pass a callback. 
And maybe Modal's 'onShow' in 'showExpandable'

Reply:
Fixed